### PR TITLE
Fixes #19076 - use new runcible call for single-consumer applicability

### PR DESCRIPTION
**Pull request by beav**
The previous consumer regen call would 'pin' to a single worker[0].

Instead, use an API call to Pulp that can be spread across multiple workers.

Note that this requires https://github.com/Katello/runcible/pull/189 to work.

[0] https://pulp.plan.io/issues/1173#note-12

### CODE REVIEW

File: app/lib/actions/pulp/consumer/generate_applicability.rb [hunk 1]
```diff
@@ -7,7 +7,11 @@ class GenerateApplicability < Pulp::AbstractAsyncTask
         end
 
         def invoke_external_task
-          pulp_extensions.consumer.regenerate_applicability_by_ids(input[:uuids])
+          if input[:uuids].length == 1
+            pulp_resources.consumer.regenerate_applicability_by_id(input[:uuids].first)
+          else
+            pulp_extensions.consumer.regenerate_applicability_by_ids(input[:uuids])
+          end
```
Comment (ehelms) [inline app/lib/actions/pulp/consumer/generate_applicability.rb]: Test possible for this conditional?

**Comment by ehelms** [comment]
runcible-1.11.0 has been released to rubygems, packaging PR open -- https://github.com/Katello/katello-packaging/pull/408

File: app/lib/actions/pulp/consumer/generate_applicability.rb (hunk 1 above)
Comment (beav) [inline app/lib/actions/pulp/consumer/generate_applicability.rb]: tests have been added

**Comment by beav** [comment]
@jlsherrill  @ehelms  green jenkins, but hound is barking at a ghost. should be ok to review at this point.

**Pull request merged**